Examples/MotionManager/MotionManager.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Included the resolve file for the example...I think that's what we want? But if not I can remove/gitignore.